source/common/filter/tcp_proxy.cc

```diff
@@ -38,7 +38,9 @@ TcpProxyConfig::Route::Route(
 
 TcpProxyConfig::TcpProxyConfig(const envoy::api::v2::filter::network::TcpProxy& config,
                                Server::Configuration::FactoryContext& context)
-    : stats_(generateStats(config.stat_prefix(), context.scope())) {
+    : stats_(generateStats(config.stat_prefix(), context.scope())),
+      max_connect_attempts_(
```

Comment:
Can we use PROTOBUF_GET_WRAPPED_OR_DEFAULT here?

Reply:
Didn't know about that one.  Yes, that'll be much nicer.